lib/command_runner.dart

```diff
@@ -320,7 +320,7 @@ abstract class Command<T> {
   /// The parsed argument results for this command.
   ///
   /// This will be `null` until just before [Command.run] is called.
-  ArgResults? get argResults => _argResults;
+  ArgResults get argResults => _argResults!;
```

Comment:
I suspect I should drop this change from the PR, and if we re-add it later, do something like - #224 - throw a StateError if this is null when called.

Reply:
I would separate this out into a different PR at least - I don't have a strong opinion on the change though.